modules/Microsoft.AAD/DomainServices/readme.md

```diff
@@ -8,6 +8,7 @@ This template deploys Azure Active Directory Domain Services (AADDS).
 - [Parameters](#Parameters)
 - [Considerations](#Considerations)
 - [Outputs](#Outputs)
+- [Cross-referenced modules](#Cross-referenced-modules)
```

Comment:
Debatable topic: Do you think we should find a better/easier word for this:

- Module also uses:
- Depends on:
- Module leverages:
- Disclaimer: This module relies on the following modules from the library:

Reply:
'Cross-referenced' was the term we used until now. I did add a disclaimer to the section itself (underneath the header).
Ain't easy to find a good name, that is meaningful and does not imply something wrong. For example, 'Depends On' isn't ideal, as this conflicts with our reference to resource dependencies.

Reply:
+1 for cross-referenced modules, maybe because I'm used to it. 
If we keep that I'd suggest to rename the utility from `Get-LinkedModuleList` to `Get-CrossReferencedModuleList`

Reply:
Renamed